proto/sequencerblockapis/astria/sequencerblock/v1alpha1/block.proto

```diff
@@ -86,10 +86,11 @@ message Deposit {
   // the address on the destination chain which
   // will receive the bridged funds
   string destination_chain_address = 5;
-  // the hash of the transaction which initiated the deposit
-  string source_transaction_hash = 6;
+  // the transaction ID of the source action for the deposit, consisting
+  // of the transaction hash.
+  astria.primitive.v1.TransactionId transaction_id = 6;
   // index of the deposit's source action within its transaction
-  uint32 source_transaction_index = 7;
+  uint32 index_of_action = 7;
```

Comment:
Maybe this?
```suggestion
  uint64 position_in_source_transaction  = 7;
```
Comment on `uint64` vs `uint32`: this is to make your life easier: `astria-core` enforces that it only compiles on 64 bit architectures, so you can safely convert from `usize` to `uint64` without having to worry. The size on the wire is the same (for any reasonable numbers) due to varint encoding.